src/control/cmd/daos_server/auto_test.go

```diff
@@ -199,31 +211,46 @@ func TestDaosServer_Auto_confGen(t *testing.T) {
 	e0 := control.MockEngineCfg(0, 2, 4).WithTargetCount(18).WithHelperStreamCount(4)
 	e1 := control.MockEngineCfg(1, 1, 3).WithTargetCount(18).WithHelperStreamCount(4)
 	exmplEngineCfgs := []*engine.Config{e0, e1}
+
+	engRsvdGiB := 2 /* calculated based on 18 targets */
+	sysRsvdGiB := storage.DefaultSysMemRsvd / humanize.GiByte
+	ramdiskGiB := storage.MinRamdiskMem / humanize.GiByte
+	// Nr hugepages expected with 18 targets * 2 engines * 512 pages-per-target.
+	defNrHugepages := 18 * 2 * 512
+	tmpfsHugeMemGiB := (humanize.MiByte * 2 * defNrHugepages) / humanize.GiByte
+	// Total mem to meet requirements 38GiB hugeMem, 2GiB per engine rsvd, 6GiB sys rsvd, 4GiB
+	// per engine RAM-disk.
+	tmpfsMemTotalGiB := humanize.GiByte * (tmpfsHugeMemGiB + (2 * engRsvdGiB) + sysRsvdGiB +
+		(2 * ramdiskGiB) + 1 /* add 1GiB buffer */)
+	tmpfsEngineCfgs := []*engine.Config{
+		control.MockEngineCfgTmpfs(0, ramdiskGiB, control.MockBdevTier(0, 2, 4)).
+			WithTargetCount(18).WithHelperStreamCount(4),
+		control.MockEngineCfgTmpfs(1, ramdiskGiB, control.MockBdevTier(1, 1, 3)).
+			WithTargetCount(18).WithHelperStreamCount(4),
+	}
+
 	metadataMountPath := "/mnt/daos_md"
 	controlMetadata := storage.ControlMetadata{
 		Path: metadataMountPath,
 	}
 	// Nr hugepages expected with 18+1 (extra MD-on-SSD sys-xstream) targets * 2 engines * 512
 	// pages-per-target.
-	tmpfsNrHugepages := 19 * 2 * 512
-	tmpfsHugeMemGiB := (humanize.MiByte * 2 * tmpfsNrHugepages) / humanize.GiByte
-	tmpfsEngRsvdGiB := 2 /* calculated based on 18 targets */
-	tmpfsSysRsvdGiB := storage.DefaultSysMemRsvd / humanize.GiByte
-	tmpfsRamdiskGiB := storage.MinRamdiskMem / humanize.GiByte
+	mdonssdNrHugepages := 19 * 2 * 512
+	mdonssdHugeMemGiB := (humanize.MiByte * 2 * mdonssdNrHugepages) / humanize.GiByte
```

Comment:
mdonssd should be bumpy case: mdOnSSD

Reply:
will address if I have to rePush

Reply:
What's the difference between bumpycase and camelcase?

Reply:
they are the same

Reply:
Will fix in a follow-up PR